README.rst

```diff
@@ -0,0 +1,262 @@
+Please check the `CHANGELOG.md <https://github.com/rl-institut/multi-vector-simulator/blob/master/CHANGELOG.md>`__ for past updates and changes.
+
+*Disclaimer*: As the MVS is still under development, changes might still occur in the code as well as code structure.
+If you want to try the MVS, please make sure to check this project regularly.
+
+For advanced programmers: You can also use the ``dev`` branch that includes the latest updates and changes.
+You find the changelog `HERE <https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md>`__.
+
+========================
+Getting started with MVS
+========================
+
+If you are interested to try out the code, please feel free to do so! In case that you are planning to use it for a specific or a larger-scale
+project, we would be very happy if you would get in contact with us, eg. via creating a github issue.
+Maybe you have ideas that can help the MVS move forward? Maybe you noticed a bug that we can resolve?
+
+We are still working on including a readthedocs for the MVS. Some information on this tool and code is already
+available `here <https://multi-vector-simulator.readthedocs.io/en/stable/>`__ (stable version,
+latest developments `here <https://multi-vector-simulator.readthedocs.io/en/latest/>`__).
+
+Rights: `Reiner Lemoine Institut (Berlin) <https://reiner-lemoine-institut.de/en/>`_
+
+The multi-vector simulator (MVS) allows the evaluation of local sector-coupled energy systems that include the energy carriers electricity, heat and/or gas. The MVS has three main features:
+
+- Firstly, an analysis of the current energy system, which can be set up automatically from a choice of components, including its costs and performance parameters.
+- Near-future investments into power generation and storage assets can be optimized aiming at a least-cost supply of electricity and heat.
+- Future energy supply scenarios that integrate emerging technologies helping to meet sustainability goals and decrease adverse climate effects, e.g. through high renewable energy shares or sector-coupling technologies, can be evaluated.
+
+The tool is being developed within the scope of the H2020 project E-LAND (Integrated multi-vector management system for Energy isLANDs, project homepage `here <https://elandh2020.eu/>`__). A graphical user interface for the MVS will be integrated.
+
+*Latest release*: Check the `latest release <https://github.com/rl-institut/multi-vector-simulator/releases/tag/v0.1.1>`__, and includes the working code of the MVS using json as an input. It is not validated and test coverage is still 0%. Please check the `CHANGELOG.md <https://github.com/rl-institut/multi-vector-simulator/blob/master/CHANGELOG.md>`__ for past updates and changes.
+
+*Disclaimer*: As the MVS is still under development, changes might still occur in the code as well as code structure.
+If you want to try the MVS, please make sure to check this project regularly.
```

Comment:
```suggestion
```
Duplicate

Reply:
Removed, thanks for pointing this out. Guess I was tired and didn't pay proper attention. Apologies.